### PR TITLE
README docs

### PR DESCRIPTION
**Pull request by mathias**
[Rendered](https://github.com/heroku/plexy/blob/mathias-readme-documentation/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,11 +18,102 @@ If [available in Hex](https://hex.pm/docs/publish), the package can be installed
...
+  2. Require Plexy plugs in your application:
 
     ```elixir
-    def application do
-      [applications: [:plexy]]
+    defmodule MyRouter do
+      use Plug.Router
+
+      plug Plexy.RequestId
+      plug Instrumentor
```
Comment (Adovenmuehle) [inline README.md]: think I'd go with `plug Plexy.Instrumentor` just to be more specific

**Comment by Adovenmuehle** [comment]
LGTM

**Comment by mathias** [comment]
@Adovenmuehle Yeah, I missed `Plexy.Instrumentor` the first time through.

**Comment by Adovenmuehle** [comment]
Hehe yeah, I think I submitted a comment about it right as you pushed up your commit to fix it.

**Pull request merged**